.buildkite/nightly-benchmarks/scripts/run-performance-benchmarks.sh

```diff
@@ -194,9 +194,11 @@ run_latency_tests() {
 
     # check if there is enough GPU to run the test
     tp=$(echo "$latency_params" | jq -r '.tensor_parallel_size')
-    if [ "$ON_CPU" == "1" ];then
-      if [[ $numa_count -lt $tp ]]; then
-        echo "Required tensor-parallel-size $tp but only $numa_count NUMA nodes found. Skip testcase $test_name."
+    pp=$(echo "$latency_params" | jq -r '.pipeline_parallel_size')
+    world_size=$(($tp*$pp))
```

Comment:
Perhaps we should move these statements into the ```ON_CPU```, as GPU tests don't set ```pipeline_parallel_size``` fields.

Reply:
make senses. changed them accordingly.